neuro_san/service/main_loop/server_main_loop.py

```diff
@@ -136,6 +136,9 @@ def prepare_args(self) -> ArgumentParser:
                                                            DEFAULT_HTTP_SERVER_MONITOR_INTERVAL_SECONDS)),
                                 help="Http server resources monitoring/logging interval in seconds "
                                      "0 means no logging")
+        arg_parser.add_argument("--mcp_enable", type=str,
+                                default=os.environ.get("AGENT_MCP_ENABLE", "false"),
+                                help="'true' if MCP protocol service should be enabled")
```

Comment:
Default is False - we will not enable MCP service unless we specify that explicitly.